### PR TITLE
Ontology keywords

### PR DESCRIPTION
**Pull request by hunterhector**
Update the ontology keywords to the new version. And modify the example data to work with the new ontology.

### CODE REVIEW

**Comment by hunterhector** [comment]
Apr-17-2020 13-03-11
Encounter this problem when playing with the new data, not sure whether this is an old problem or a bug introduced by this PR.

**Comment by hunterhector** [comment]
Metadata tab broke with this PR.

**Comment by hunterhector** [comment]
> Metadata tab broke with this PR.

Fixed in ee15c18

**Comment by hunterhector** [comment]
> Apr-17-2020 13-03-11
> Encounter this problem when playing with the new data, not sure whether this is an old problem or a bug introduced by this PR.

Fixed in 8380047

**Comment by hunterhector** [comment]
Additional bugs are caused by the new data, temporarily fixed by ec0f052

File: simple-backend/README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ A simple backend server application that manage documents and persist data.
   - `. venv/bin/activate` use virtual env
 - make sure django is installded, or install it by:
   - `python -m pip install Django`
+- To use the example db, remove the .example extension:
+  - `mv db.sqlite3.backup db.sqlite3`
```
Comment (GingerBear) [inline simple-backend/README.md:20]: Don't think we need this. In `./start-dev.sh` it's already doing this

```
if [ ! -f db.sqlite3 ]; then
    cp db.sqlite3.example db.sqlite3
    echo 'db.sqlite3 created'
fi
```

File: src/nlpviewer/components/AnnotationLabel.tsx [hunk 2]
```diff
@@ -67,13 +67,12 @@ function AnnotationLabel({
                 <>
                   <span className={style.annotation_attr_label}>{attrKey}</span>
                   <span className={style.annotation_attr_value}>
-                    {annotation.attributes[attrKey]}
+                  {/*Is this the floating window? */}
```
Comment (GingerBear) [inline src/nlpviewer/components/AnnotationLabel.tsx:70]: yes

File: src/nlpviewer/components/TextDetail.tsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ export default function TextDetail({
     title: 'metadata',
     body: () => (
       <div>
-        <Attributes attributes={attributes} />
+        <Attributes attributes={attributes['py/state']} />
```
[the remark this answers is not in the document]
Comment (hunterhector) [reply src/nlpviewer/components/TextDetail.tsx]: Resolved in a7d6a38

**Pull request merged**